README.md

```diff
@@ -62,28 +58,28 @@ module.exports = {
 Add a `--player` option to the command to include the client-side Javascript "player" boilerplate to the new widget module as well.
 
 ```bash
-apos create-widget tabs --player
+apos add widget tabs --player
 ```
 
 ## Create a piece
 To bootstrap the necessary files and basic configuration for a new Apostrophe piece type, run the following command from within your Apostrophe project's root directory:
 
 ```bash
-apos create-piece vegetables
+apos add piece vegetables
 ```
 
 Then remember to register `'vegetables': {}` in `apps.js` above.
 
-If you run the `create-piece` command with the `--pages` flag, the command will also set up a corresponding pieces-pages module with your new piece type. Similarly, you can run the `create-piece` command with the `--widgets` flag, which will also set up a corresponding pieces-widgets module along with your new piece type. These flags can be used together or separately.
+If you run the `add piece` command with the `--page` flag, the command will also set up a corresponding pieces-pages module with your new piece type. Similarly, you can run the `add piece` command with the `--widget` flag, which will also set up a corresponding pieces-widgets module along with your new piece type. These flags can be used together or separately.
```

Comment:
We should bias toward the 3.x singular language rather than the legacy 2.x plural language, piece-page is better

Reply:
👍 Totally reworking the README here will be a task soon. Or simply direct them to the docs, which can be nice.